.changeset/witty-insects-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/kv': patch
```

Comment:
```suggestion
'@vercel/kv': minor
```

This should at least be a minor release -- obviously we're in beta, so breakings are acceptable in minors, but I would never put a breaking change in a patch release...

Reply:
oh weird, I was sure I selected the `minor` in the changeset